src/main/java/com/epam/esm/dto/LoginRequest.java

```diff
@@ -0,0 +1,4 @@
+package com.epam.esm.dto;
+
+public record LoginRequest(String username, String password) {
```

Comment:
This is not LoginRequest, this is LoginRequestDTO

Reply:
my fault, fixed